client/dashboard/sites/performance/subtitle.tsx

```diff
@@ -1,47 +1,57 @@
 import { __experimentalText as Text, Button } from '@wordpress/components';
 import { createInterpolateElement } from '@wordpress/element';
 import { __, sprintf } from '@wordpress/i18n';
-import { useFormattedTime } from '../../components/formatted-time';
+import { useTimeSince } from '../../components/time-since';
+
+const REPORT_REFRESH_THRESHOLD_HOURS = 48;
+
+function isReportOlderThan( reportTimestamp: string, hours: number ): boolean {
+	const now = new Date();
+	const reportDate = new Date( reportTimestamp );
+
+	if ( isNaN( reportDate.getTime() ) ) {
+		return false;
+	}
+
+	return now.getTime() - reportDate.getTime() > hours * 60 * 60 * 1000;
+}
 
 export default function Subtitle( {
 	timestamp,
-	timezoneString,
-	gmtOffset,
 	onClick,
 }: {
 	timestamp: string | undefined;
-	timezoneString?: string;
-	gmtOffset?: number;
 	onClick: () => void;
 } ) {
-	const formattedTime = useFormattedTime(
-		timestamp ?? '',
-		{
-			dateStyle: 'long',
-			timeStyle: 'short',
-		},
-		timezoneString,
-		gmtOffset
-	);
+	const timeSince = useTimeSince( timestamp ?? '' );
 
 	if ( ! timestamp ) {
 		return <Text variant="muted">{ __( 'Testing your site may take around 30 seconds.' ) }</Text>;
 	}
 
+	if ( isReportOlderThan( timestamp, REPORT_REFRESH_THRESHOLD_HOURS ) ) {
+		return createInterpolateElement(
+			sprintf(
+				/* translators: %s: relative time since last test run */
+				__(
+					'Last test ran <b>%s</b>. Test again if your site has changed. <button>Test again</button>'
```

Comment:
Nit: I think it might read better if we reduce the duplication:

```
'Last test ran <b>%s</b>. <button>Test again</button> if your site has changed.
```

or

```
'Last test ran <b>%s</b>. If your site has changed, <button>test again</button>'
```

Reply:
Great suggestion.

Reply:
I like the second version because it keeps "test again" in the same place. Do we have any guidance on whether that needs a final "." or not? I don't see it in the [docs](https://github.com/Automattic/wp-calypso/blob/trunk/client/dashboard/docs/typography-and-copy.md). It feels weird without the period at the end.